README.md

```diff
@@ -61,6 +61,7 @@ following environment variables:
 | `PORTUNUS_SLAPD_TLS_CA_CERTIFICATE` | *(optional)* | *Required* when a TLS certificate is given. The full chain of CA certificates which has signed the TLS certificate, *including the root CA*. |
 | `PORTUNUS_SLAPD_TLS_DOMAIN_NAME` | *(optional)* | *Required* when a TLS certificate is given. The domain name for which the certificate is valid. `portunus-server` will use this domain name when connecting to the LDAP server. |
 | `PORTUNUS_SLAPD_TLS_PRIVATE_KEY` | *(optional)* | *Required* when a TLS certificate is given. The path to the private key belonging to the TLS certificate. |
+| `PORTUNUS_POSIX_ACCOUNT_REGEX` | *(optional)* | Configures a username regex. |
```

Comment:
1. Since changing the regex results in usernames that are not POSIX-conformant, we should not imply as much in the name (e.g. `PORTUNUS_USER_ACCOUNT_REGEX`). This note also applies to the implementation, where the concept of "POSIX-conformant" is mixed with the configurable regex that makes it non-compliant.
2. The default behavior should be documented.